app/tasks/raster_tile_cache_assets/raster_tile_cache_assets.py

```diff
@@ -62,6 +64,15 @@ async def raster_tile_cache_asset(
         ).replace("{tile_id}.tif", "tiles.geojson")
     ]
 
+    # Create a generic calc string when a raster tile cache is being
+    # created from a multi-band raster tile set.
+    band_count = source_asset.creation_options["band_count"]
+    if band_count > 1:
+        bands_string = str(list(string.ascii_uppercase[:band_count])).replace("'", "")
+        calc: Optional[str] = f"np.ma.array({bands_string})"
```

Comment:
instead of requiring this np.ma.array always in the calc string, should we just automatically wrap the calc string in np.ma.array?

Reply:
Hmm. It's not ALWAYS required (the details are not worth going into), but I can't think of any harm to wrapping the calc string in an extra np.ma.array, and it would mean the user wouldn't have to remember to do so. Sounds good, though I think the change should be made in pixetl. When I've done that I'll modify any relevant calc strings in the data API code.

Reply:
gotcha, we can do this in a separate PR then